### PR TITLE
Add pointer to the super-heroes workshop

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by holly-cummins** [comment]
Good idea!

**Review by holly-cummins — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 This is a sample application demonstrating Quarkus features and best practices. The application allows superheroes to fight against supervillains. The application consists of several microservices, communicating either synchronously via REST or asynchronously using Kafka. All the data used by the applications are [on the `characterdata` branch](https://github.com/quarkusio/quarkus-super-heroes/tree/characterdata) of this repository.
 
+This is **NOT** the workshop with the step-by-step instructions. If you are looking for the Quarkus Super Heroes workshop, you can find it at https://github.com/quarkusio/quarkus-workshops/tree/main/quarkus-workshop-super-heroes.
```
Comment (edeandrea) [inline README.md:20]: Would https://quarkus.io/quarkus-workshops/super-heroes/ be a better link for the workshop? @holly-cummins / @xstefank ?

Comment (xstefank) [reply README.md:20]: TBH, I have no preference. So I can change it if you think it's better.

Comment (edeandrea) [reply README.md:20]: I'll let @holly-cummins weigh in.

Comment (holly-cummins) [reply README.md:20]: If people want the formatted instructions, Eric's link is better. And, thinking about it, people are most likely to want the formatted instructions. 

We should probably put a link from the formatted instructions back to the repo, for PRs + bug reports, but that's a separate (bigger) thing.

Comment (edeandrea) [reply README.md:20]: When you get the formatted instructions there is already a link back to the repo:

image

**Review by edeandrea — APPROVED** [review]

**Comment by edeandrea** [comment]
Thanks @xstefank !

**Pull request merged**